opennms-webapp-rest/src/test/java/org/opennms/web/rest/v1/config/JmxDataCollectionConfigResourceIT.java

```diff
@@ -84,17 +84,49 @@ public void testJmxConfig() throws Exception {
 
         assertThat(config.getJmxCollectionCount(), greaterThanOrEqualTo(6));
 
+        assertEquals("activemq", config.getJmxCollection("activemq").getName());
```

Comment:
We can remove these. The purpose of this test is to verify that we can retrieve the JMX configuration via REST. We don't need to assert all of the groups - it's a pain to maintain otherwise.